src/ApiManagement/ApiManagement/help/New-AzApiManagementOperation.md

```diff
@@ -84,7 +84,7 @@ Accept wildcard characters: False
 ```
 
 ### -ApiRevision
-Identifier of API Revision. This parameter is optional. If not specified, the operation will be attached to the currently active api revision.
+Identifier of API Revision. This parameter is optional. If not specified, the operation will be attached to the currently active api revision.Non-current revision has ;rev=n as a suffix where n is the revision number.
```

Comment:
Thanks for the contribution. The expression has typo `;rev=n`. Welcome to resubmit the PR have the typo fixed if you want to continue your work. We close this PR now since no update for a long time.